README.md

```diff
@@ -128,4 +128,7 @@ Apache 2.0, See [LICENSE](LICENSE) for more information.
 
 # Data and telemetry
 
-The Red Hat Dependency Analytics Extension for Visual Studio Code collects anonymous [usage data](Telemetry.md) and sends it to Red Hat servers to help improve our products and services. Read our [privacy statement](https://developers.redhat.com/article/tool-data-collection) to learn more. This extension respects the `redhat.elemetry.enabled` setting which you can learn more about at https://github.com/redhat-developer/vscode-commons#how-to-disable-telemetry-reporting
\ No newline at end of file
+The Red Hat Dependency Analytics Extension for Visual Studio Code collects anonymous [usage data](Telemetry.md) and sends it to Red Hat servers to help improve our products and services. Read our [privacy statement](https://developers.redhat.com/article/tool-data-collection) to learn more. This extension respects the `redhat.elemetry.enabled` setting which you can learn more about at https://github.com/redhat-developer/vscode-commons#how-to-disable-telemetry-reporting
+
+
+> **Disclaimer** : Dependency Analytics does not analyse dev/test dependencies associated with the Stack.
```

Comment:
IMHO, this should be moved to the top, probably after the short intro about the extension.

@prashbnair WDYT?

Reply:
Yes, it should be placed on top.

Reply:
ack